pipelines/skylab/snM3C/snM3C.wdl

```diff
@@ -24,18 +26,37 @@ workflow snM3C {
         Int compress_level = 5
         Int batch_number
 
+        String cloud_provider
+
     }
 
     # version of the pipeline
-    String pipeline_version = "2.0.1"
+    String pipeline_version = "2.0.2"
+
+    String snM3C_docker = "m3c-yap-hisat:1.0.0-2.2.1"
```

Comment:
We had discussed splitting the large snM3C docker up in the future. Will we just move this top-level input for snM3C_docker into the individual tasks?

Reply:
We would include the individual dockers here and then change which docker is specified for each task